src/aparna/codingExam1/ReturnIndicesOfTwoNumbers.java

```diff
@@ -0,0 +1,42 @@
+/*Program 1 : 
+Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to the target.
+
+Input: nums = [2,11,15,7,4], target = 9
+
+Output: [0,3]
+
+Output: Because nums[0] + nums[3] == 9, we return [0, 3].
+ */
+package aparna.codingExam1;
+
+public class ReturnIndicesOfTwoNumbers {
+
+	public int[] returnIndicesofTWoNum(int[] inputNumber, int targetNumber) {
+
+		int[] outputArr = new int[inputNumber.length];
+
+		for (int outerIndex = 0; outerIndex < inputNumber.length; outerIndex++) {
+
+			for (int innerIndex = 1; innerIndex < inputNumber.length; innerIndex++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files